config/deploy.rb

```diff
@@ -28,7 +28,7 @@
 set :deploy_via, :remote_cache
 set :git_shallow_clone, 1
 set :use_sudo, false
-set :rvm_ruby_string, '2.0.0'
+set :rvm_ruby_string, '2.1.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.